groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -354,6 +354,16 @@ groups:
       - prignanov@vmware.com
       - vrgf2003@gmail.com
 
+  - email-id: k8s-infra-staging-capi-ipam-ic@kubernetes.io
+    name: k8s-infra-staging-capi-ipam-ic
+    description: |-
+      ACL for staging CAPI IPAM Provider Incluster
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - jakob.schrettenbrunner@telekom.de
```

Comment:
If this affects permissions in GCP it should probably be my identical @gmail.com address instead.

Reply:
It does, can you send a "suggestion" comment with the change? So it will be committed together here

Reply:
```suggestion
      - jakob.schrettenbrunner@gmail.com
```